e2e/rbd.go

```diff
@@ -487,6 +516,13 @@ var _ = Describe("RBD", func() {
 						go func(w *sync.WaitGroup, n int, p v1.PersistentVolumeClaim, a v1.Pod) {
 							name := fmt.Sprintf("%s%d", f.UniqueName, n)
 							wgErrs[n] = createPVCAndApp(name, f, &p, &a, deployTimeout)
```

Comment:
you will need to check for errors here, to verify that the PVC and App have been created correctly